languages/python/exercises/concept/conditionals/.docs/after.md

```diff
@@ -124,7 +125,7 @@ else:
 >>> Hello,World
 ```
 
-In this example we did not evaluvate if condition like this `if str == "":`. Even then its understood by python that if a string is empty evaluvate it to `False` and evaluvate to `True` otherwise.
+In this example we did not evaluate if condition like this `if str == "":`. Even then it's understood by python that if a string is empty evaluate it to `False` and evaluate to `True` otherwise.
```

Comment:
```suggestion
In this example we did not evaluate if condition like this `if str == "":`. Even then it's understood by Python that if a string is empty evaluate it to `False` and evaluate to `True` otherwise.
```